### ISSUE TITLE
[Feature Request] [Editor] "fast up"/"fast down" for int values

### ISSUE THREAD

**Issue by Favna** [thomasnet-mc/EdiZon#12]
***This is mostly in lieu of a proper custom value input editor***

The new changes to the editor are great already but we should get "fast up" and "fast down" features for integer values. For example when the value of Hyrule Warriors rupees is 268159 and you want to go to 999999 your best bet is to go down till 0 then let to roll around to 999999 but we're still talking over 2 million times 1 down which takes a LOT of time. 

A fast up and fast down feature could be `hold L + up/down` or some other combination which will have it go somewhere in the vicinity of 10x faster. 

On a sidenote, when you guys find time for a custom editor for int values please make the keyboard just have numbers ala the numpad you have when entering a number on your phone. No need for letters when you know it's just numbers.

**Comment by WerWolv98** [comment]
Uhm. We already included speedup so that after a few seconds of holding the stick down, it speeds up to 20/step. Is that still too slow?

**Comment by Favna** [comment]
ye it is.. [here is a recording on streamable](https://streamable.com/8irmf)

**Comment by WerWolv98** [comment]
Okay you're right. But even 10 times faster might still take ages. How about holding the stick down and going left/right will increase the value by 1000? There aren't a lot of buttons left that we can use in the editor so would that be a solution?

**Comment by Favna** [comment]
Yeah sure that would work too. Maybe give various button combinations for jumps by 100, 1000, 1000000 etc? Not sure how far you guys want to push it. I'm thinking something like this:

- `up/down` = 100
- `L+up/down` = 1000
- `R+up/down` = 10000
- `L+R+up/down` = 1000000

**Comment by WerWolv98** [comment]
The problem with that would be, all of those buttons already have other functions. Only buttons that aren't set so far are Y and both of the sticks. ZR and ZL aren't set either but they switch through the games and accounts if not in the editor so I don't want to cause confusion by using them. I could do a little hint somewhere on how big the steps are right now and you can switch through them by pressing one of sticks. Other options would be to automatically determent the step size by calculating a value with (maxValue - minValue) or to be able to set your own step size in the config (optionally)

**Comment by Favna** [comment]
Of those suggestions I think I'd most prefer a combination of the step size, or if none is provided (so as fallback) the calculation of the min and max values.

**Comment by WerWolv98** [comment]
So manually specify the step size and if none is set, calculate it?

**Comment by Favna** [comment]
correct
